src/masternode.h

```diff
@@ -244,6 +244,19 @@ class CMasternode : public masternode_info_t
                 nActiveStateIn == MASTERNODE_WATCHDOG_EXPIRED;
     }
 
+    static std::string GetMNLevelStr(int level)
+    {
+        switch(level)
+        {
+            default: return  "-";
+
+            // 1 based level string for user facing visualisations
+            case 0:  return "1";
+            case 1:  return "2";
+            case 2:  return "3";
```

Comment:
consider:

case 0:
case 1:
case 2:
  return std::to_string(level+1);

Reply:
I did consider that but wanted the UI visualisation to be obvious if another level was added and this logic wasn't followed through.